package.json

```diff
@@ -24,7 +23,8 @@
     "lint-node": "eslint '**/*.js' --ignore-pattern 'client/**/*'",
     "start": "node server.js",
     "webpack": "NODE_ENV='production' webpack -p --config webpack.config.js",
-    "test": "jest"
+    "test-root": "jest --testPathIgnorePatterns='/functions'",
```

Comment:
Hmm... so this is conceptually incorrect as it is using the root-package "jest" installation to run the tests in the /function package which breaks the hermeticity.

travis should instead do npm install and npm run while inside the functions directory... thinking about this more, we might want to do in an independent CL.

Reply:
I don't have a ton of experience with travis but can we have a different config file for it in the functions dir?

Maybe for now the "test-root" can be renamed to "test" and we just remove the "test-functions" script? That way we can get this change in and then figure out how to work in testing for the functions dir

Reply:
@awong-dev Do you think I should leave the /functions folder as is and revert it back to the old jest test? As far as I know, no one is currently working on the cloud function unit tests just yet. We could get to to when we are working on them later on.

Reply:
It's fine what you have for now. We can fix it later.